automation/project/Dependencies.scala

```diff
@@ -13,7 +13,7 @@ object Dependencies {
   val workbenchGoogle: ModuleID = "org.broadinstitute.dsde.workbench" %% "workbench-google" % workbenchGoogleV excludeAll excludeWorkbenchModel
   val excludeWorkbenchGoogle = ExclusionRule(organization = "org.broadinstitute.dsde.workbench", name = "workbench-google_2.11")
 
-  val workbenchServiceTestV = "0.7-847c3ff"
+  val workbenchServiceTestV = "0.9-e97a335-SNAP"
```

Comment:
Will replace this after https://github.com/broadinstitute/workbench-libs/pull/125 is merged and a new non-SNAP version is published.